src/Libraries/CoreNodeModels/CustomSelectionItem.cs

```diff
@@ -0,0 +1,165 @@
+using System;
+using System.ComponentModel;
+using System.Windows.Input;
+using Newtonsoft.Json;
+
+namespace CoreNodeModels
+{
+    /// <summary>
+    /// Represents an Enum item
+    /// </summary>
+    public class CustomSelectionItem
+    {
+        public string Name { get; set; }
+
+        public string Value { get; set; }
+
+        public CustomSelectionItem()
+        {
+
+        }
+    }
+
+
+    public class CustomSelectionItemViewModel : INotifyPropertyChanged, IDataErrorInfo
+    {
+        private readonly CustomSelectionItem customSelectionItem;
+
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        #region Properties
+
+        [JsonIgnore]
```

Comment:
is this type serialized somewhere? That is surprising to me - and if it's not serialized, why do you need the the json attributes?